test/WebJobs.Script.Tests.E2E/GeneralEndToEndTests.cs

```diff
@@ -199,50 +201,8 @@ public async Task ServiceBus_Node_DoesNotExhaustConnections()
                 }
             }
 
-            QueueDescription queueDescription = manager.GetQueue("node");
-            Assert.Equal(i * j, queueDescription.MessageCountDetails.ActiveMessageCount);
-        }
-
-        [Fact(Skip = "Proxy not yet enabled.")]
-        [TestTrace]
-        public async Task FileExtension()
-        {
-            using (var client = CreateClient())
-            {
-                HttpResponseMessage response = await client.GetAsync($"test.txt");
-
-                string content = await response.Content.ReadAsStringAsync();
-                _fixture.Assert.Equals("200", response.StatusCode.ToString("D"));
-                _fixture.Assert.Equals("test", content);
-            }
-        }
-
-        [Fact(Skip = "Proxy not yet enabled.")]
-        [TestTrace]
-        public async Task RootCheck()
-        {
-            using (var client = CreateClient())
-            {
-                HttpResponseMessage response = await client.GetAsync("/");
-
-                string content = await response.Content.ReadAsStringAsync();
-                _fixture.Assert.Equals("200", response.StatusCode.ToString("D"));
-                _fixture.Assert.Equals("Root", content);
-            }
-        }
-
-        [Fact(Skip = "Proxy not yet enabled.")]
-        [TestTrace]
-        public async Task LocalFunctionCall()
```

Comment:
Did you mean to delete these tests? Your comment says this is enabling proxy tests but I don't see these being moved anywhere...

Reply:
Ahh, okay, that makes sense.